src/i18n/en.json

```diff
@@ -184,6 +184,12 @@
     "ariaListTitlePhones": "EN: Antamasi puhelinnumerot",
     "ariaListTitleEmails": "EN: Antamasi sähköpostiosoitteet",
     "ariaListTitleAddresses": "EN: Antamasi osoitteet",
+    "ariaSetPrimaryAddressWithValue": "Set {{value}} as primary address",
+    "ariaSetPrimaryEmailWithValue": "Set {{value}} as primary email",
+    "ariaSetPrimaryPhoneWithValue": "Set {{value}} as primary phone",
+    "ariaRemoveAddressWithValue": "Remove address {{value}}",
+    "ariaRemoveEmailWithValue": "Remove email {{value}}",
+    "ariaRemovePhoneWithValue": "Remove phone {{value}}",
```

Comment:
Should the term "phone number" be used instead of "phone"? Both terms seem to be in use, but feels like "phone number" better refers to the _number_, whereas just "phone" refers to the device.

Reply:
Both seems to be used. I changed new and old ones to use "phone number"